examples/lightning_base.py

```diff
@@ -217,28 +199,26 @@ def add_model_specific_args(parser, root_dir):
 
 
 class LoggingCallback(pl.Callback):
+    @rank_zero_only
     def on_validation_end(self, trainer: pl.Trainer, pl_module: pl.LightningModule):
         logger.info("***** Validation results *****")
-        if pl_module.is_logger():
-            metrics = trainer.callback_metrics
-            # Log results
-            for key in sorted(metrics):
-                if key not in ["log", "progress_bar"]:
-                    logger.info("{} = {}\n".format(key, str(metrics[key])))
+        metrics = trainer.callback_metrics
+        # Log results
+        for key in sorted(metrics):
+            if key not in ["log", "progress_bar"]:
+                logger.info("{} = {}\n".format(key, str(metrics[key])))
```

Comment:
If the goal is to only log metrics on rank_zero use:

`rank_zero_info('my message')`